test-crawler/extractor/extractor.go

```diff
@@ -301,3 +353,15 @@ func makeID(filePath string, funcName string, behavior string) string {
 	hash := md5.Sum([]byte(fmt.Sprintf("%s_%s_%s", filePath, funcName, behavior)))
 	return string(hex.EncodeToString(hash[:]))
 }
+
+func parseContentForPublicFunctions(content string, cursor *sitter.TreeCursor) ([]c.FunctionAnnotation, error) {
```

Comment:
The method is not named properly (it does not return only public functions, but all functions), and it does not have a GoDoc comment.

Reply:
I think that unexported functions does not need comments by default for GoDoc, but I will add simple one explaining what that functions does.

Reply:
As far as I know there will be linter error for functions that are exported and have no comment. But a good practice would be for all the functions both exported and unexported to have a simple comment.

Reply:
> As far as I know there will be linter error for functions that are exported and have no comment. But a good practice would be for all the functions both exported and unexported to have a simple comment.

Correct. I would prefer to have a simple comment for each function we add, even if the function is unexported.